recipe/meta.yaml

```diff
@@ -32,22 +32,25 @@ requirements:
     - python    # [win]
     - cmake >=3.1
     - zlib 1.2*
+    - gcc  # [not win]
+    - libgfortran  # [not win]
```

Comment:
Drop the `libgcc` run dependency, but keep `libgfortran`. That's ideally how we want to ship this. We may need some more tweaks to get it there.